### PR TITLE
One release

### PR DESCRIPTION
**Pull request by haroldsphinx**
- use one workflow to manage all releases

### CODE REVIEW

File: .github/workflows/client-release.yml [hunk 1]
```diff
@@ -2,11 +2,8 @@ name: "Dali-chachacha Community Release"
 
 on:
   push: 
-    tags: 
-      - 'dali-*'
-
-env:
-  CHAIN: "dali"
+    branches: 
+      - 'one-release'
```
Comment (seunlanlege) [inline .github/workflows/client-release.yml:6]: Why not just `release`?

**Review by seunlanlege — CHANGES_REQUESTED** [review]
I also don't see the check runtime workflow

File: .github/workflows/client-release.yml (hunk 1 above)
Comment (haroldsphinx) [reply .github/workflows/client-release.yml:6]: it's still WIP...thats my test branch

**Comment by haroldsphinx** [comment]
> I also don't see the check runtime workflow

its on a separate workflow https://github.com/ComposableFi/composable/blob/main/.github/workflows/runtime-check.yml

**Review by seunlanlege — APPROVED** [review]

**Review by nimboya — APPROVED** [review]

**Pull request merged**